examples/overlay-warnings/README.md

```diff
@@ -0,0 +1,11 @@
+# Overlay
+
+```shell
+node ../../bin/webpack-dev-server.js --open
+```
+
+## What should happen
+
+The script should open the browser and show a heading with "Example: overlay with warnings".
+
+In `app.js`, make a syntax error. The page should now refresh and show a full screen error overlay that shows the syntax error.
```

Comment:
Could you document how to trigger a warning here? This text only tells me how to trigger an error. Would be handy for testing, since I can't come up with something that triggers a warning out the top of my head right now :).